client/src/routes.js

```diff
@@ -1,7 +1,7 @@
 import Home from "./pages/Home";
 import NeedHelp from "./pages/NeedHelp";
 import OfferHelp from "./pages/OfferHelp";
-import About from "./pages/About";
+import AboutUs from "./pages/AboutUs";
```

Comment:
Later we could have an _index.js_ at the _/pages_ exporting all the pages, and then, in the routes, import them like this:

`
import {
  Home,
  NeedHelp,
  OfferHelp,
  AboutUs
} from "./pages";
`

Reply:
Noted! I think there will be plenty of refactoring to be done.